src/_includes/layouts/layout.liquid

```diff
@@ -3,9 +3,8 @@
 <head>
   <meta charset="utf-8" />
   <style>
-    {%- postcss -%}
-      {% include entry.css %}
-    {%- endpostcss -%}
+    {% capture css %}{% include entry.css %}{% endcapture %}
+    {{ css postcss }}
```

Comment:
Sorry, I need to fetch a laptop to verify, but if you’re converting to a filter, shouldn’t this be `{{ css | postcss }}`?

Reply:
Doesn't help, still ends up with `{}`.

Anyway, docs also says pipe with `|`. Updated here https://github.com/pdehaan/11ty-postcss/pull/1/commits/22932c7acd3f0b8709a21c09e630e6693eeaaa57.

I've ran the code both with and w/o the `|` and the postcss filter gets executed regardless, so maybe it's optional. Result is still `{}`.